Mathlib/Analysis/InnerProductSpace/Projection.lean

```diff
@@ -986,6 +986,10 @@ open Module
 
 variable [FiniteDimensional 𝕜 K]
 
+@[simp]
+theorem topologicalClosure_eq : K.topologicalClosure = K :=
```

Comment:
It doesn't say what it should be equal to, and without `self`, it's not obvious that the hypothesis might be `[FiniteDimensional 𝕜 E]`.
```suggestion
theorem topologicalClosure_eq_self : K.topologicalClosure = K :=
```